temporian/core/operators/propagate.py

```diff
@@ -40,7 +40,7 @@ def __init__(
 
         # TODO: Constraint the type of features supported in "add_event".
 
-        if event.sampling != to.sampling:
+        if event.sampling is not to.sampling:

```

Comment:
Why do we want to check that the reference is the same and not use the equals? Using the equals is better because we could have two different references with the same schema. Is there any reason for this change?